src/containers/Cluster/Cluster.scss

```diff
@@ -8,13 +8,29 @@
 
     width: 100%;
     height: 100%;
-    padding: 0 var(--cluster-side-padding);
+
+    &__sticky-wrapper {
+        padding: 0 var(--cluster-side-padding);
+    }
+
+    &__dashboard {
+        position: sticky;
+        left: 0;
+
+        padding: 0 var(--cluster-side-padding);
+    }
+
+    &__content {
+        width: calc(100% - var(--cluster-side-padding));
```

Comment:
It seems that this is the last evil, but essential. For now our scroll container doesn't have paddings, so sticky elements stick to it.
I found this solution better than passing to all tables inside Cluster page classes with `left: 20px`

Reply:
What if you add left padding to cluster scroll container, and right padding to all cluster components (dashboard, tabs, tabs content)?